### ISSUE TITLE
[1.6.10-beta02] [iOS] Gradle re-sync keep trying to update project.pbxproj incorrectly

### ISSUE THREAD

**Issue by AlexanderEggers** [JetBrains/compose-multiplatform#4704]
**Describe the bug**
After updating to **1.6.10-beta02**, I noticed that almost every gradle re-sync attempts to make updates to the `project.pbxproj` file. The potential changes are sometimes around removing the `[CP] Copy Pods Resources` or `[CP] Embed Pods Frameworks` build phases. If I would allow these changes, my ios project would stop working - so ignoring these changes actually works fine at this moment.

**Affected platforms**
- iOS

**Versions**
- *Kotlin version*: 2.0.0-RC1
- *Compose Multiplatform version*: 1.6.10-beta02

**To Reproduce**
Steps and/or the code snippet to reproduce the behavior:
1. Ensure that the project is setup with a shared folder which is exposed as a cocoapod static framework.
2. Do a gradle sync
3. See project.pbxproj changes

**Expected behavior**
No incorrect changes to the `project.pbxproj` file which would break ios projects.

**Comment by AlexanderEggers** [comment]
Update:
I had a closer look at the issue and the issue seems to be much more significant in regards to iOS. It doesn't actually automatically detect the composeResource folder anymore. I did a test and the relevant Pods-iosApp-resources-{configuration}-{input/output}-files.xcfilelist are not longer created for some reasons. If I add these manually, then everything seems to work but otherwise it just cannot find any shared resources anymore which might explain why the build system tries to remove the relevant build phase.

**Comment by AlexanderEggers** [comment]
Random thought.. the root cause could be around the fact that I named my compose (shared) code framework different than the folder name which is contains the code.

So my project folder structure is:
- androidApp
- iosApp
- shared

The framework name for the "shared" folder is called "CommonLib". For the most part that seems to work but the podspec definition `spec.resources` keeps pointing to `['build/compose/ios/shared/compose-resources']` which is not correct. It should be `['build/compose/ios/CommonLib/compose-resources']`. So I just renamed my shared framework to "shared" and finally the relevant `xcfilelist` files are generated again. It still tried to remove the resource build phase - so that seems to be a separate is to the framework naming.

I hope some of these observations help to understand my issue.
